repose-aggregator/components/filters/cors-filter/src/test/scala/org/openrepose/filters/cors/CorsFilterTest.scala

```diff
@@ -708,7 +737,9 @@ class CorsFilterTest extends FunSpec with BeforeAndAfterEach with Matchers with
         servletRequest.setScheme("http")
         servletRequest.setServerName(null)
         servletRequest.setServerPort(-1)
-        forwardedHostHeaders foreach { servletRequest.addHeader(CommonHttpHeader.X_FORWARDED_HOST, _) }
+        forwardedHostHeaders foreach {
+          servletRequest.addHeader(CommonHttpHeader.X_FORWARDED_HOST, _)
+        }
```

Comment:
If you wanted, you could keep this on one line like:
```
forwardedHostHeaders.foreach(servletRequest.addHeader(CommonHttpHeader.X_FORWARDED_HOST, _))
```

Reply:
Updated.